src/openrct2/drawing/Drawing.cpp

```diff
@@ -87,6 +87,8 @@ uint8_t gTextPalette[0x8] = {
     0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
 };
 
+bool gPaintForceRedraw;
```

Comment:
Shouldn't this be initialised to false? Looks like it will just contain whatever was in that memory before.